chipy_org/templates/homepage.html

```diff
@@ -230,7 +230,8 @@ <h3>This Month's Topics</h3>
                       {% if topic.experience_level %}
                       Experience Level: {{ topic.get_experience_level_display }}<br />
                       {% endif %}
-                      <span  class="more">{{ topic.description }}</span>
+                      {{ topic.description|striptags|truncatewords:80|safe }}<br />
```

Comment:
if you can sanitize the html from the user could you use
`{{ topic.description|truncatewords_html:80|safe }}<br />`
instead of 
`{{ topic.description|striptags|truncatewords:80|safe }}<br />`
the output of striptags is not guaranteed to be "safe"